The nixos test fails on aarch64-linux.

```pytb
Traceback (most recent call last):
  File "/nix/store/8dzxyl9f33i5kfn56akd54jkizr4rpdl-python3.12-aiorun-2025.1.1/lib/python3.12/site-packages/aiorun/__init__.py", line 223, in new_coro
    await coro
  File "/nix/store/wzfvpa7yk0iv8hr1nqysdzx7y4pwbwhp-music-assistant-2.5.2/lib/python3.12/site-packages/music_assistant/__main__.py", line 214, in start_mass
    await mass.start()
  File "/nix/store/wzfvpa7yk0iv8hr1nqysdzx7y4pwbwhp-music-assistant-2.5.2/lib/python3.12/site-packages/music_assistant/mass.py", line 138, in start
    self.aiozc = AsyncZeroconf(ip_version=IPVersion.V4Only, interfaces=InterfaceChoice.Default)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8xcqa1507mb9r3ccww42ni6zwiy49a86-python3.12-zeroconf-0.147.0/lib/python3.12/site-packages/zeroconf/asyncio.py", line 171, in __init__
    self.zeroconf = zc or Zeroconf(
                          ^^^^^^^^^
  File "/nix/store/8xcqa1507mb9r3ccww42ni6zwiy49a86-python3.12-zeroconf-0.147.0/lib/python3.12/site-packages/zeroconf/_core.py", line 181, in __init__
    listen_socket, respond_sockets = create_sockets(interfaces, unicast, ip_version, apple_p2p=apple_p2p)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8xcqa1507mb9r3ccww42ni6zwiy49a86-python3.12-zeroconf-0.147.0/lib/python3.12/site-packages/zeroconf/_utils/net.py", line 459, in create_sockets
    add_multicast_member(cast(socket.socket, listen_socket), interface)
  File "/nix/store/8xcqa1507mb9r3ccww42ni6zwiy49a86-python3.12-zeroconf-0.147.0/lib/python3.12/site-packages/zeroconf/_utils/net.py", line 350, in add_multicast_member
    listen_socket.setsockopt(socket.IPPROTO_IP, socket.IP_ADD_MEMBERSHIP, _value)
OSError: [Errno 19] No such device
2025-05-10 03:20:32.665 INFO (MainThread) [aiorun] Entering shutdown phase.
2025-05-10 03:20:32.665 INFO (MainThread) [aiorun] Executing provided shutdown_callback.
2025-05-10 03:20:32.665 INFO (MainThread) [root] shutdown requested!
2025-05-10 03:20:32.666 INFO (MainThread) [music_assistant] Stop called, cleaning up...
2025-05-10 03:20:32.666 INFO (MainThread) [aiorun] Waiting for executor shutdown.
2025-05-10 03:20:32.666 INFO (MainThread) [aiorun] Shutting down async generators
2025-05-10 03:20:32.666 INFO (MainThread) [aiorun] Closing the loop.
2025-05-10 03:20:32.667 INFO (MainThread) [aiorun] Leaving. Bye!
2025-05-10 03:20:32.667 INFO (MainThread) [aiorun] Reraising unhandled exception
2025-05-10 03:20:32.667 ERROR (MainThread) [root] Uncaught exception
Traceback (most recent call last):
  File "/nix/store/wzfvpa7yk0iv8hr1nqysdzx7y4pwbwhp-music-assistant-2.5.2/bin/.mass-wrapped", line 9, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/wzfvpa7yk0iv8hr1nqysdzx7y4pwbwhp-music-assistant-2.5.2/lib/python3.12/site-packages/music_assistant/__main__.py", line 220, in main
    run(
  File "/nix/store/8dzxyl9f33i5kfn56akd54jkizr4rpdl-python3.12-aiorun-2025.1.1/lib/python3.12/site-packages/aiorun/__init__.py", line 374, in run
    raise pending_exception_to_raise
  File "/nix/store/8dzxyl9f33i5kfn56akd54jkizr4rpdl-python3.12-aiorun-2025.1.1/lib/python3.12/site-packages/aiorun/__init__.py", line 276, in run
    shutdown_callback(loop)
  File "/nix/store/wzfvpa7yk0iv8hr1nqysdzx7y4pwbwhp-music-assistant-2.5.2/lib/python3.12/site-packages/music_assistant/__main__.py", line 205, in on_shutdown
    loop.run_until_complete(mass.stop())
  File "/nix/store/gbml27jdc722w5srjw6mw28v08wi0xbf-python3-3.12.10/lib/python3.12/asyncio/base_events.py", line 691, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/nix/store/wzfvpa7yk0iv8hr1nqysdzx7y4pwbwhp-music-assistant-2.5.2/lib/python3.12/site-packages/music_assistant/mass.py", line 205, in stop
    await self.streams.close()
          ^^^^^^^^^^^^
AttributeError: 'MusicAssistant' object has no attribute 'streams'

--- a/nixos/modules/services/audio/music-assistant.nix
+++ b/nixos/modules/services/audio/music-assistant.nix
@@ -69,6 +69,9 @@ in
       description = "Music Assistant";
       documentation = [ "https://music-assistant.io" ];
 
+      after = [ "network-online.target" ];
+      wants = [ "network-online.target" ];
+
       wantedBy = [ "multi-user.target" ];
 
       environment = {

--- a/pkgs/by-name/mu/music-assistant/frontend.nix
+++ b/pkgs/by-name/mu/music-assistant/frontend.nix
@@ -7,12 +7,12 @@
 
 buildPythonPackage rec {
   pname = "music-assistant-frontend";
-  version = "2.14.8";
+  version = "2.14.9";
   pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-YUir/YBBbggsQUh5b6qSG5fpAa25jJmgcSsf0EZ8rhw=";
+    hash = "sha256-UEGRZBegoAnls5xAyVgjisD0B8nu8kXp1XHI4A114lw=";
   };
 
   postPatch = ''

--- a/pkgs/by-name/mu/music-assistant/package.nix
+++ b/pkgs/by-name/mu/music-assistant/package.nix
@@ -48,14 +48,14 @@ assert
 
 python.pkgs.buildPythonApplication rec {
   pname = "music-assistant";
-  version = "2.5.0";
+  version = "2.5.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "music-assistant";
     repo = "server";
     tag = version;
-    hash = "sha256-yugtL3dCuGb2OSTy49V4mil9EnfACcGrYCA1rW/lo+4=";
+    hash = "sha256-RkbU2MqQ7XSv7f6gvgS0AZ8jy63fUAomC41dEk8qyOI=";
   };
 
   patches = [

--- a/pkgs/by-name/mu/music-assistant/providers.nix
+++ b/pkgs/by-name/mu/music-assistant/providers.nix
@@ -1,7 +1,7 @@
 # Do not edit manually, run ./update-providers.py
 
 {
-  version = "2.5.0";
+  version = "2.5.2";
   providers = {
     airplay = ps: [
     ];

--- a/pkgs/development/python-modules/plexapi/default.nix
+++ b/pkgs/development/python-modules/plexapi/default.nix
@@ -11,7 +11,7 @@
 
 buildPythonPackage rec {
   pname = "plexapi";
-  version = "4.16.1";
+  version = "4.17.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
@@ -20,7 +20,7 @@ buildPythonPackage rec {
     owner = "pkkid";
     repo = "python-plexapi";
     tag = version;
-    hash = "sha256-ncinm2wtFx2lzsDTsK+D4rz4hVhcwcK2VkL6osNUftY=";
+    hash = "sha256-/CYWoEHmev5e5ZmlaBms1zclRwzvugAuG2JXzK9MeZA=";
   };
 
   build-system = [ setuptools ];

